content/tutorials/manage-the-rippled-server/configuration/propose-amendments.md

```diff
@@ -0,0 +1,111 @@
+
+- Abstract/Introduction
+- Specifications
+- Considerations
+
+
+<!-- This hasn't been updated since 2021; is this step from the README valid? -->
+When a standard moves to a folder + file(s) in the Code section of this repository, it will be added to the standards.toml file: https://github.com/XRPLF/XRPL-Standards/blob/master/standards.toml
+
+<!-- How much is enough discussion? The README mentions creating an issue to reference in the PR, but this doesn't seem to be happening in practice. -->
+After sufficient input from the community, create a corresponding **Issue** with the finalized standards draft in `XRPLF:develop`.
+
+
+## Create an Amendment
+
+Any changes that affect **transaction processing** requires an amendment. This applies to:
+
+- Features
+- Enhancements
+- Bug Fixes
+
+If your change doesn't require an amendment, you can skip this step.
+
+1. Add the name of your amendment as a string to `rippled/src/ripple/protocol/impl/Feature.cpp` to define and initialize a  `uint256` variable. For example:
+
+```cpp
+REGISTER_FEATURE(FlowCross,                     Supported::yes, DefaultVote::yes);
+REGISTER_FEATURE(CryptoConditionsSuite,         Supported::yes, DefaultVote::no);
+REGISTER_FIX    (fix1513,                       Supported::yes, DefaultVote::yes);
+```
+
+  - `Supported` parameter should be set to `no` initially.
+  - `DefaultVote` parameter should be set to `yes` for bug fixes. Work with the community to decide the default vote for features and enhancements.
+
+2. Navigate to `rippled/src/ripple/protocol/Feature.h` and modify these items:
+
+  - Increment the `numFeatures` counter.
+
+  - Add an `extern uint256 const` variable declaration. For example:
+
+```cpp
+extern uint256 const featureFlowCross;
+extern uint256 const featureCryptoConditionsSuite;
+extern uint256 const fix1513;
+```
+
+3. Navigate to `https://github.com/XRPLF/rippled/tree/master/src/ripple/app/tx/impl` and make your changes. Be sure to include an if/else condition to keep old behavior if it's not enabled.
+
+## Develop and Test the Code
+
+1. Create a fork or branch in the [`rippled` repository](https://github.com/XRPLF/rippled) to develop your code.
+
+2. As you develop your amendment, run unit and integration tests. Running a server in *Stand-alone* mode is useful for testing changes, but you may want to stand up a private network for extensive changes.
+
+3. When development is complete, update the `Supported` parameter to `yes` in `Feature.cpp`.
+
+4. Create a **Pull Request** on `XRPLF:develop` and link to the corresponding issue you created.
+
+5. After the PR is approved by XRP Ledger maintainers, your code is merged to `develop`. Additional testing can be done on Devnet.
+
+**Note:**
+  - The `DefaultVote` parameter is now locked.
+  - If problems are found with the amendment, you must restart the process of making fixes and submitting a new PR. You can change the defaultvote in the new PR.
```

Comment:
These notes are missing from the concepts doc in "Coding and Deployment". With regards to the second bullet point, there are also a praxis for rolling up amendments in a new versioned amendment, in case bugs are found pre inclusion in mainnet (`NonFungibleTokensV1_1`)